### ISSUE TITLE
Convert ocp-indent beautifier to an Executable with custom path support

### ISSUE THREAD

**Issue by AllanWang** [Glavin001/atom-beautify#1862]
This is a follow up on the [original PR](https://github.com/Glavin001/atom-beautify/pull/934#issuecomment-330618261) that adds support for OCaml

Attempting to beautify the file in Windows 10 results in the following:

```log
See ocp-indent installation instructions at https://github.com/Glavin001/atom-beautify#beautifiers or go to https://www.typerex.org/ocp-indent.html
Your program is properly installed if running 'where.exe ocp-indent' in your CMD prompt returns an absolute path to the executable.
Hide Stack Trace
Error: Could not find 'ocp-indent'. The program may not be installed.
    at Function.Executable.commandNotFoundError (file:///C:/Users/[user]/.atom/packages/atom-beautify/src/beautifiers/executable.coffee:276:14)
    at HybridExecutable.Executable.commandNotFoundError (file:///C:/Users/[user]/.atom/packages/atom-beautify/src/beautifiers/executable.coffee:268:18)
    at file:///C:/Users/[user]/.atom/packages/atom-beautify/src/beautifiers/executable.coffee:196:22
    at tryCatcher (C:\Users\[user]\.atom\packages\atom-beautify\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (C:\Users\[user]\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:512:31)
    at Promise._settlePromise (C:\Users\[user]\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (C:\Users\[user]\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (C:\Users\[user]\.atom\packages\atom-beautify\node_modules\bluebird\js\release\promise.js:689:18)
    at Async._drainQueue (C:\Users\[user]\.atom\packages\atom-beautify\node_modules\bluebird\js\release\async.js:133:16)
    at Async._drainQueues (C:\Users\[user]\.atom\packages\atom-beautify\node_modules\bluebird\js\release\async.js:143:10)
    at Async.drainQueues (C:\Users\[user]\.atom\packages\atom-beautify\node_modules\bluebird\js\release\async.js:17:14)
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

This is expected, as

```
where.exe ocp-indent
```
Does not find anything.

I've installed `ocp-indent` successfully through `opam`, but it is located in `C:\OCaml64\home\[user]\.opam\4.03.0+mingw64\bin` and can only be accessed through the OCaml64 terminal.

# Checklist

I have:
- [x] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [x] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [ ] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

**Comment by szeck87** [comment]
I'm trying to get the docker image up and running for ocp-indent.  I need to test it out on Windows, but if it works it should be an easy install and run out of the box.  I'll provide an update soon.

**Comment by Glavin001** [comment]
What needs to happen is setup `ocp-indent` as an `Executable` so custom executable path option is supported.

**Comment by Glavin001** [comment]
Published to v0.32.0
